rotkehlchen/data/all_assets.json

```diff
@@ -14858,6 +14858,66 @@
         "symbol": "yWETH",
         "type": "ethereum token"
     },
+    "yvDAI": {
```

Comment:
Why are you adding something in this file? Don't do that. This file is not used anywhere. It's legacy file kept only for the DB upgrade purposes.

Reply:
This was a change made before the globaldb